nemo_curator/utils/script_utils.py

```diff
@@ -472,6 +451,7 @@ def parse_gpu_dedup_args(description: str) -> argparse.ArgumentParser:
 
         # Set default device to GPU for dedup
         argumentHelper.parser.set_defaults(device="gpu")
+        argumentHelper.parser.set_defaults(set_torch_to_use_rmm=False)
```

Comment:
@ayushdg ,  This is what you wanted me switch off right ?

Reply:
That's right!